libs/s25main/network/GameClient.cpp

```diff
@@ -1719,7 +1719,7 @@ std::string GameClient::FormatGFTime(const unsigned gf) const
     using std::chrono::duration_cast;
 
     // In Sekunden umrechnen
-    seconds numSeconds = duration_cast<seconds>(gf * framesinfo.gf_length);
+    seconds numSeconds = duration_cast<seconds>(gf * (std::chrono::milliseconds)SPEED_GF_LENGTHS[referenceSpeed]);
```

Comment:
Just for consistency: 
```suggestion
    seconds numSeconds = duration_cast<seconds>(gf * std::chrono::milliseconds(SPEED_GF_LENGTHS[referenceSpeed]));
```